.github/workflows/wasm.yml

```diff
@@ -16,14 +15,64 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Setup rust toolchain
-        uses: dtolnay/rust-toolchain@master
+      - name: Setup Nix
+        uses: cachix/install-nix-action@v20
         with:
-          toolchain: 1.66.0
+          nix_path: nixpkgs=channel:nixos-22.11
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install wasm-pack
-        run: cargo install wasm-pack
+      - name: Build wasm package
+        run: |
+          nix build -L .#wasm --extra-experimental-features nix-command --extra-experimental-features flakes
```

Comment:
Are `--extra-experimental-features nix-command --extra-experimental-features flakes` necessary? They should be removed

Reply:
Removed